stack/app/rds.tf

```diff
@@ -9,6 +10,7 @@ module "rds-kms-key" {
 
   # Optional
   deletion_window_in_days = var.kms_deletion_window_in_days
+  multi_region            = var.rds_multi_region_kms_key
```

Comment:
What happened if this is a rds replica, and no `var.rds_master_db_kms_key_arn` is provided? 

Should we do some sort of check to make sure user is using the replica key 

```
multi_region = var.rds_is_read_replica ? false : var.rds_multi_region_kms_key
```
or is it okay to have a non replica key as read-replica rds's kms key?

Reply:
good point. The read-replica must use the master KMS key, unless the master DB does not use encryption at rest.